tests/acceptance/features/apiAuthApp/token.feature

```diff
@@ -129,4 +129,25 @@ Feature: create auth-app token
         "minItems": 0,
         "maxItems": 0
       }
+      """
+
+
+  Scenario: try to create auth-app token without expiry
+    When user "Alice" tries to create app token with expiration time "" using the auth-app API
+    Then the HTTP status code should be "400"
+    And the content in the response should match the following content:
+      """
+      error parsing expiry. Use e.g. 30m or 72h
+
+      """
+
+  @env-config
+  Scenario: admin tries to create auth-app token for other users without expiry
+    Given the config "AUTH_APP_ENABLE_IMPERSONATION" has been set to "true"
+    When the administrator tries to create app token for user "Alice" with expiration time "" using the auth-app API
```

Comment:
this somehow went unnoticed. Please, use `the administrator` to denote system admin. So for ocis admin user use this step
```suggestion
    When user "Admin" tries to create app token for user "Alice" with expiration time "" using the auth-app API
```

Reply:
and you can remove step `the administrator tries to create app token...`